src/installer/prepare-artifacts.proj

```diff
@@ -145,10 +145,6 @@
         <RelativeBlobPath>$(InstallersRelativePath)workloads/%(Filename)%(Extension)</RelativeBlobPath>
         <PublishFlatContainer>true</PublishFlatContainer>
       </ItemsToPush>
-
-      <!-- Source build intermediated packages will be pushed and signed by the sourcebuild leg. -->
```

Comment:
How come this changed

Reply:
Now that we aren't pushing a copy of the source-build artifacts into the IntermediateArtifacts artifact, we can remove this code that filters it out.